components/Autocomplete.tsx

```diff
@@ -21,9 +21,9 @@ import {
   useDisclosure,
   useMultiStyleConfig,
 } from '@chakra-ui/react'
-import { TriangleDownIcon } from '@chakra-ui/icons'
 import useOutsideClickHandler from 'hooks/useOutsideClickHandler'
```

Comment:
Can we use `useOutsideClick ` from chakra-ui ?

Reply:
No. `useOutsideClick` cannot handle outside click from two components(for example `Input` and `Popover`).